traitsui/qt4/tabular_editor.py

```diff
@@ -923,8 +923,10 @@ def columnResized(self, index, old, new):
         This affects the column widths when not using auto-sizing.
         """
         if not self._is_resizing:
-            if self._user_widths is None:
-                self._user_widths = [None] * len(self._editor.adapter.columns)
+            num_columns = len(self._editor.adapter.columns)
+            if self._user_widths is None or \
+                    len(self._user_widths) != num_columns:
```

Comment:
```suggestion
            if (self._user_widths is None 
                    or len(self._user_widths) != num_columns):
```
Style nit: we prefer to do continuations with parentheses rather than `\`